index.html

```diff
@@ -0,0 +1,1848 @@
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/ad0c4e912f.js"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+      integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"
+      integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"
+      integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy"
+      crossorigin="anonymous"
+    ></script>
+    <title>Newsweek</title>
+  </head>
+  <body>
+    <header class="position-relative">
+      <div class="navbar nav sticky-top">
+        <div class="d-none d-lg-inline-block">
+          <div class="text-white font-weight-bold">
+            <img class="wicon" src="images/weather.png" alt="weather" /> 17°
+            Caraguatatuba, BR >
+          </div>
+          <div class="text-white">Wed, Jun 03,2020</div>
+        </div>
+        <a href="#"
+          ><img
+            class="navbar-brand newsweek-logo"
+            src="images/newsweek.png"
+            alt="newsweek"
+        /></a>
+        <div
+          class="login-div d-inline-flex align-items-center justify-content-lg-end"
+        >
+          <a href="#"><span class="fas fa-user d-md-none"></span></a>
+          <h3 class="login d-none d-md-inline">LOGIN</h3>
+          <button class="btn btn-warning d-md-none">SUBSCRIBE</button>
+          <button class="btn btn-warning btn-warning2 d-none d-md-inline">
+            SUBSCRIBE >
+          </button>
+          <nav class="d-lg-none d-flex align-items-center">
+            <button
+              class="navbar-toggler"
+              type="button"
+              data-toggle="collapse"
+              data-target="#navbarToggleExternalContent"
+              aria-controls="navbarToggleExternalContent"
+              aria-expanded="false"
+              aria-label="Toggle navigation"
+            >
+              <span class="fas fa-bars"></span>
+            </button>
+          </nav>
+        </div>
+      </div>
+      <nav class="navbot nav position-relative d-none d-lg-flex">
+        <ul class="nav align-items-center">
```

Comment:
- [ ] Kindly fix this section for medium and small devices. It should not be displayed.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83824060-1aeb6e80-a69b-11ea-8d27-44649ff0597a.png) | ![image](https://user-images.githubusercontent.com/57414789/83824044-0dce7f80-a69b-11ea-95b1-213dae66efbc.png)